trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/generate_airfoil_transitions.py

```diff
@@ -0,0 +1,100 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
+# compute_airfoil_transition.py
+# 
+# Created:  Mar 2021, R. Erhard
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry import import_airfoil_geometry 
+from SUAVE.Plots.Geometry_Plots import plot_airfoil
+import numpy as np
+
+def generate_airfoil_transition(a1, a2, space, nairfoils, save_file=False, save_filename="Transition", npts=50):
+    """ Takes in two airfoils and their locations and interpolates
+    between them to establish a transition region.
+    
+    Assumptions: Linear geometric transition between airfoils
+    
+    Source: None
+    
+    Inputs:
+    a1                 first airfoil                                [ airfoil ]
+    a2                 second airfoil                               [ airfoil ]
+    y1                 spanwise location of first airfoil           [ m ]
+    y2                 spanwise location of second airfoil          [ m ]
+    n                  number of transitional airfoils              [ unitless ]
+    space              spanwise distance for transition to occur    [ m ]
+    
+    """
+    
+    # import airfoil geometry for the two airfoils
+    airfoil_geo_files = [a1, a2]
+    a_geo = import_airfoil_geometry(airfoil_geo_files,npts)
+    
+    # identify x and y coordinates of the two airfoils
+    x_upper = a_geo.x_upper_surface
+    y_upper = a_geo.y_upper_surface
+    x_lower = a_geo.x_lower_surface
+    y_lower = a_geo.y_lower_surface
+    
+    # setup spanwise stations for transitional airfoils
+    z = np.linspace(0,space,nairfoils)
```

Comment:
I don't understand this space variable.

For example, in the regression it has airfoils from 0 to 0.2 with 4 points so: 0, 0.067, .133, and 0.2

But then  zairfoils is 0 to space. So what does space do? I set space to 1.0 and it gave me the same result.

Reply:
Removed. I originally had an additional output to identify the locations of the newly generated airfoils for a fixed transition region.